charts/rancher-monitoring-crd/102.0.1+up40.1.2/values.yaml

```diff
@@ -0,0 +1,17 @@
+# Default values for rancher-monitoring-crd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    psp:
+      enabled: false
+    systemDefaultRegistry: ""
+
+image:
+  repository: rancher/shell
+  tag: v0.1.18
```

Comment:
Can you please use version [v0.1.19](https://github.com/rancher/shell/releases/tag/v0.1.19) instead?

Reply:
I'll update this when fixing the commit formats

Reply:
bumped `rancher/shell` version